tests/robot-tests/tests/admin/bau/release_content.robot

```diff
@@ -79,35 +79,35 @@ Add Useful information related page link to release
     user clicks button    Create link
 
 Add secondary statistics
-    user waits until page does not contain element    ${SECONDARY_STATS_TABLE_TAB_SELECTOR}
+    user waits until page does not contain element    //*[@id="${SECONDARY_STATS_TABLE_TAB_SELECTOR}"]
```

Comment:
I think it's worth changing the variable name here to something like `SECONDARY_STATS_TABLE_TAB_ID`.

Reply:
I think you can just use `id:${SECONDARY_STATS_TABLE_TAB_ID}` here rather than using xpath. There are other places to make this change too.